src/microbe_stage/FloatingChunk.cs

```diff
@@ -221,6 +248,27 @@ private void VentCompound(Vector3 pos, string compound, float amount)
             compound, amount * Constants.CHUNK_VENT_COMPOUND_MULTIPLIER, pos);
     }
 
+    /// <summary>
+    ///   Handles the dissolving effect for the chunks when they run out of compounds.
+    /// </summary>
+    private void HandleDissolving(float delta)
+    {
+        // Disable collisions
+        CollisionLayer = 0;
+        CollisionMask = 0;
+
+        var material = (ShaderMaterial)chunkMesh.MaterialOverride;
```

Comment:
Are the iron chunks now setup as well so that they have per scene materials? Otherwise this doesn't work correctly.

Reply:
I did enable the `Local To Scene` flag on the `IronMaterial.material` file.

Reply:
I did check that myself, I meant to delete this comment instead of posting it...